dat/tech.xml

```diff
@@ -635,6 +647,11 @@
  <tech name="Criminal Outfits">
   <item>Camouflage Burster</item>
  </tech>
+ <tech name="Mining Outfits">
+  <item>Asteroid Scanner</item>
+  <item>Medium Weapon License</item>
+  <item>Milspec Impacto-Plastic Coating</item>
```

Comment:
This should be a Dvaered-only outfit.

Reply:
I added Hulls Cargo into the same place that the SK cargo hulls already were, does this not work like I think it does? (i.e. categories can't be added into other categories to include their contents as well?)

Reply:
> I added Hulls Cargo into the same place that the SK cargo hulls already were, does this not work like I think it does? (i.e. categories can't be added into other categories to include their contents as well?)

Bobbens is referring to the impacto-plastic coating

Reply:
fixed